src/mrinufft/operators/interfaces/gpunufft.py

```diff
@@ -502,6 +502,19 @@ def uses_sense(self):
         """Return True if the Fourier Operator uses the SENSE method."""
         return self.raw_op.uses_sense
 
+    def update_smaps(self,new_smaps):
+        """Update pinned smaps from new_smaps.
+
+        Parameters
+        ----------
+        new_smaps: np.ndarray
+            the new sensitivity maps
+
+        """
+        smaps_ = new_smaps.T.reshape(-1, new_smaps.shape[0])
+        np.copyto(self.raw_op.pinned_smaps, smaps_)
+        self.smaps = new_smaps
+
```

Comment:
this won't be enough for gpuNUFFT, you also need to make changes in the `RawGpuNUFFT` class, and do the proper management with respect to the smaps.

Reply:
Umm why wont this be enough? The pointer to the pinned_smaps is pointed. As long as the data changes, it should be fine right?